### DISCUSSION TITLE
Media queries

### DISCUSSION THREAD

**Discussion by ajalapraise** [tailwindlabs/tailwindcss#12088] [Ideas]
I am not sure if anybody has talked about this but I Think there should be a way we can use the Media queries like sm, md, lg with just a single declaration and not writing it every time we want to style for that screen size

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)

**Reply by ajalapraise** [reply]
Thank you Very much
